docs/HACKING.md

```diff
@@ -244,35 +242,34 @@ You'll need to fetch the current MAAS main:
 git clone http://git.launchpad.net/maas
 ```
 
-And then build MAAS and set up some sampledata:
+And then build and install a MAAS snap:
 
 ```shell
 cd maas
+sudo snap install maas-test-db
+sudo apt install make
 make install-dependencies
-make
-make syncdb
-make sampledata
+git config --file=.gitmodules submodule.src/maasui/src.branch main
+git submodule sync
+git submodule update --init --recursive --remote
+make snap-prime
+sudo snap try build/dev-snap/prime
+utilities/connect-snap-interfaces
+sudo maas init region+rack --maas-url=http://localhost:5240/MAAS --database-uri maas-test-db:///
 ```
 
-Now you should be ready to start the MAAS, you'll need to do this each time you start the Multipass instance.
+You'lll also need to create a user:
```

Comment:
You'lllll probably be fine with only two l's

Reply:
I dunno feels like it could use a couple more ls